bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -482,7 +482,7 @@ private String getStatusMessage(List<Comment> comments, List<Review> reviews, Pu
                             }
                             if (iss.get().state() != org.openjdk.skara.issuetracker.Issue.State.OPEN) {
                                 if (!pr.labels().contains("backport")) {
-                                    progressBody.append(" ⚠️ Issue is not open.");
+                                    progressBody.append(" ⚠️ Issue is not open.\n");
                                 }
                                 continue;
```

Comment:
I know I wrote this, but another solution would be to drop the `continue` here, I don't remember why it's there. :)

Reply:
Ok, will do that than. (seems a little cleaner to me). I think that will only change whether the `if` below is evaluated. Maybe you wanted only one warning to be printed at a time?

Reply:
Yeah I guess, but then it was a bit inconsistent with the rest of the warnings anyway... So this looks good to me!